src/core/ContactShadows.tsx

```diff
@@ -57,11 +59,27 @@ export const ContactShadows = React.forwardRef(
       const blurPlane = new THREE.Mesh(planeGeometry)
       const depthMaterial = new THREE.MeshDepthMaterial()
       depthMaterial.depthTest = depthMaterial.depthWrite = false
-      depthMaterial.onBeforeCompile = (shader) =>
-        (shader.fragmentShader = shader.fragmentShader.replace(
+      depthMaterial.onBeforeCompile = (shader) => {
+        shader.uniforms = {
+          ...shader.uniforms,
+          ucolor: {
+            value: new THREE.Color(color).convertSRGBToLinear(),
```

Comment:
Am I right to call `convertSRGBToLinear` here since it is a `Color` object?

Reply:
yes... :-)